captum/attr/_core/feature_permutation.py

```diff
@@ -24,27 +24,47 @@ def _permute_feature(x: Tensor, feature_mask: Tensor) -> Tensor:
 
 class FeaturePermutation(FeatureAblation):
     r"""
-    This attribution method essentially implements the permutation feature
-    importance algorithm, as described here:
-    https://christophm.github.io/interpretable-ml-book/feature-importance.html
+    A perturbation based approach to compute attribution, which
+    takes each input feature, permutes the feature values within a batch,
+    and computes the difference in output between the original and shuffled.
```

Comment:
nit: maybe, `... and computes the difference between original and shuffled outputs for given batch. Those differences signify feature importance for the permuted features ` ?